chromium_src/BUILD.gn

```diff
@@ -290,8 +290,118 @@ source_set("browser") {
     "chrome/browser/content_settings/host_content_settings_map_factory.cc",
     "chrome/browser/content_settings/host_content_settings_map_factory.h",
 
-    "//chrome/browser/ssl/security_state_tab_helper.cc",
+    "//chrome/browser/renderer_preferences_util.cc",
+    "//chrome/browser/renderer_preferences_util.h",
+
+    "//chrome/browser/safe_browsing/chunk_range.cc",
```

Comment:
I would put these in a separate safe_browsing target and make that a dep of `browser`